app/src/main/res/layout/viewflipper_signup_step2.xml

```diff
@@ -54,7 +54,8 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginBottom="@dimen/textinput_margin"
-                android:layout_marginTop="@dimen/textinput_margin">
+                android:layout_marginTop="@dimen/textinput_margin"
+                app:passwordToggleEnabled="true">
```

Comment:
Bound namespace `app` 
add this in beginning `xmlns:app="http://schemas.android.com/apk/res-auto"`
Also update the `build.gradle` in this PR only